plugins/hydra_ray_launcher/tests/test_ray_aws_launcher.py

```diff
@@ -259,15 +259,15 @@ def manage_cluster() -> Generator[None, None, None]:
         (
             "ray_aws",
             [
-                f"hydra.launcher.ray_cluster_cfg.cluster_name={cluster_name}",
+                f"hydra.launcher.ray.cluster.cluster_name={cluster_name}",
                 "hydra.launcher.stop_cluster=False",
                 f"hydra.launcher.sync_down.source_dir={sweep_dir}/",
                 f"hydra.launcher.sync_down.target_dir={sweep_dir}",
-                f"hydra.launcher.ray_cluster_cfg.provider.key_pair.key_name=hydra_test_{cluster_name}",
-                "hydra.launcher.ray_cluster_cfg.setup_commands=[]",
-                "hydra.launcher.mandatory_install.install_commands=[]",
-                f"+hydra.launcher.ray_cluster_cfg.worker_nodes={ray_nodes_conf_override}",
-                f"+hydra.launcher.ray_cluster_cfg.head_node={ray_nodes_conf_override}",
+                f"hydra.launcher.ray.cluster.provider.key_pair.key_name=hydra_test_{cluster_name}",
+                "hydra.launcher.ray.cluster.setup_commands=[]",
+                "hydra.launcher.env_setup.commands=[]",
```

Comment:
why do you need to override the commands?

Reply:
:)
This is due to our test instance doesn't have access to pip so the setup commands would not work during integration testing.

Reply:
Gotcha. add a comment about it?

Reply:
thanks, added.